README.md

```diff
@@ -279,8 +279,17 @@ You can specify media shortcuts through the `shortcuts` option, either as a list
       - icon: mdi:dog
         type: script
         id: script.script_name
+      - name: Crooners
```

Comment:
Could we add a comment above this line? A short description like `#Trigger custom service` or similar, like the other shortcut examples.